internal/config/mount_config.go

```diff
@@ -19,7 +19,9 @@ type WriteConfig struct {
 }
 
 type LogConfig struct {
-	Severity LogSeverity `yaml:"severity"`
+	Severity  LogSeverity `yaml:"severity"`
+	LogFormat string      `yaml:"format"`
```

Comment:
is it possible to keep the property name same as yaml field name. In that case you need to refer to logConfig as mountConfig.LogConfig.format. In that case do we need to add nil check on LogConfig everytime accessed via mountConfig?

Going forward if we need to add same properties in different configs, it will be easier to refer using their sub config names.

Reply:
In golang, the zero-value of a struct type is the struct with all its elements set to their zero values so it will never be nil. 

Made the changes to use the subconfig (full path).